bind.go

```diff
@@ -39,10 +41,26 @@ type maybeLineage struct {
 
 	allv []SyntacticVersion
 
+	implens []ImperativeLens
+
+	// translation plan, tracks whether each lens is handled in cue or go
+	transplan map[lensID]bool
+
 	// The raw input value is the root of a package instance
 	// rawIsPackage bool
 }
 
+// to, from
+type lensID [2]SyntacticVersion
```

Comment:
any reason not to use a struct here? seems like it could easily be confusing which is from and which is to

Reply:
indeed! https://github.com/grafana/thema/pull/187#discussion_r1270338534